src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,33 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { findUsedById } from '../../services/findUsedById';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+import todosFromServer from '../../api/todos';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const maxId = Math.max(...todosFromServer
```

Comment:
`todosFromServer` is a static variable, so there will be calculated the same id for each new todo
you should calculate it when you add a new one, based on the actual todos array, here just set `data-id={todo.id}`